package.json

```diff
@@ -29,6 +29,8 @@
     "webpack-shell-plugin-alt": "^1.4.0"
   },
   "devDependencies": {
+    "axios": "^1.7.7",
```

Comment:
uit nieuwsgierigheid: waarom niet met fetch? Dan heb je geen extra dependency.

Reply:
Axios werkt met promises en is gemakkelijk om dan dingen uit te voeren nadat je de response binnen hebt en te chainen. Kan ook met fetch uiteindelijk, dus het antwoord is wss uit gemakzucht. Ik gebruik axios zo goed als altijd als ik ajax calls moet doen.